source/tubtrunk/lib/Controllers/statisticController.dart

```diff
@@ -7,15 +7,18 @@ class StatisticController {
   int _failedFocusTimes;
   int _succeedFocusTimes;
   int _totalFocusTimes;
+  double _averageFocusTimes;
   List<TimerRecordModel> _timerRecords;
 
   StatisticController() {
     _failedFocusTimes = 0;
     _succeedFocusTimes = 0;
     _totalFocusTimes = 0;
+    _averageFocusTimes=0;
```

Comment:
Can you please update some format where I left comments, just small adjustments?